include/xtensor/xtensor_simd.hpp

```diff
@@ -59,6 +59,12 @@ namespace xsimd
         static constexpr size_t size = 1;
     };
 
+    template <class T>
+    struct simd_batch_traits
```

Comment:
This is internal to `xsimd` and should not be defined here. You can use `reverse_simd_traits` to get the properties of a batch.